assets/js/components/notifications/cta.js

```diff
@@ -43,6 +43,13 @@ class CTA extends Component {
 			onClick,
 		} = this.props;
 
+		let message = null;
+		if ( description ) {
+			message = typeof description === 'string'
+				? <p className="googlesitekit-cta__description">{ description }</p>
+				: description;
+		}
```

Comment:
This is a bit of a nitpick but I think it would be preferable to keep the structure of the markup in place/inline as much as possible. Also changing `description` to `message` is another level of indirection which slightly less easy to read.

I would suggest replacing `{ message }` with something like this which both maintains the class for both cases and omits the need for additional temporary variables.

```js
{ ( description && typeof description === 'string' ) && (
    <p className="googlesitekit-cta__description">{ description }</p>
) }
{ ( description && typeof description !== 'string' ) && (
    <div className="googlesitekit-cta__description">{ description }</div>
) }
```

Reply:
Updated.